packages/fake-driver/test/helpers.js

```diff
@@ -1,17 +1,21 @@
 import path from 'path';
 import wd from 'wd';
+import {existsSync} from 'fs';
 
-
-const TEST_HOST = 'localhost';
+const TEST_HOST = '127.0.0.1';
 const TEST_PORT = 4774;
-const TEST_APP = path.resolve(__dirname, '..', '..', 'test', 'fixtures', 'app.xml');
+
+// XXX: what dir are we in? it depends on how we're running this test. the .xml file doesn't move, but we do!
+// we may be running with @babel/register; otherwise, we're probably in `build/test/` which _does not_ contain the xml file.
+const appFixturePath = path.join(__dirname, 'fixtures', 'app.xml');
+const TEST_APP = existsSync(appFixturePath) ? appFixturePath : path.join(__dirname, '..', '..', 'test', 'fixtures', 'app.xml');
```

Comment:
two potential fixes here

1. stop using babel 😄
2. copy the xml file into `build` during transpilation

Reply:
option #2 looks reasonable to be

Reply:
yeah this bites us in a few different places, where we have to unreasonably path out to the package root and then down into the build dir. if there's an easy way to copy fixtures during transpilation, great!

Reply:
should we create an issue to track this task?